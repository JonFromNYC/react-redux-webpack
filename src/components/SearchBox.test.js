import {shallow} from 'enzyme';
import React from 'react';
import SearchBox from './SearchBox.component';

// console.log("Shallow Test:\t",shallow(<SearchBox />).length); //length === 1
describe('<SearchBox />', ()=>{
    it('renders a SearchBox component', ()=>{
        expect(shallow(<SearchBox />)).toMatchSnapshot();
    });
})