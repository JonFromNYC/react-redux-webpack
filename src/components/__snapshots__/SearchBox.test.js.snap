// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> renders a SearchBox component 1`] = `ShallowWrapper {}`;
